OREAnalytics/test/CMakeLists.txt

```diff
@@ -32,6 +32,9 @@ target_link_libraries(orea-test-suite ${ORED_LIB_NAME})
 target_link_libraries(orea-test-suite ${OREA_LIB_NAME})
 target_link_libraries(orea-test-suite ${Boost_LIBRARIES} ${RT_LIBRARY})
 
+# avoid ABI mismatch between precompiled boost.log and this compiler run...
```

Comment:
Maybe add comment that this is fix for boost >1.73 both manual and pre-builds.

Reply:
I've added a hint for 1_73 now.